src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -4983,7 +4983,8 @@ def set_index_from_columns(
             keys
         ):
             # Error checking for missing labels is already done in frontend layer.
-            index_column_snowflake_quoted_identifiers.append(ids[0])
+            if ids:
```

Comment:
see "# Error checking for missing labels is already done in frontend layer."
what's the error did you see here? And why it happens now?

Reply:
Sorry, that was some experimentation, will remove it now